ouroboros-consensus/src/consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Some.hs

```diff
@@ -0,0 +1,131 @@
+{-# LANGUAGE DataKinds                  #-}
```

Comment:
This module is a utility, much more general than `ChainGenerators`, but relied upon heavily by it.